src/classes/CW_AudienceCSVProcessController.cls

```diff
@@ -0,0 +1,131 @@
+/**
+* File Name   :    CW_AudienceCSVProcessController.cls
+* Description :    Page Contrloer for second step in importing Audience Rewards
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 20/05/2014 Alex Panka	  Intial 
+*/
+
+public with sharing class CW_AudienceCSVProcessController {
+	public final static Integer BATCH_SIZE=2000;
+	public CW_CSV_Import_Status__c impRevStat{get;private set;}
+	
+	public boolean IsProcessing{get;private set;}
+	public ID  batchJobId{get;private set;}
+	public integer countTiks{get;set;}
+	public AsyncApexJob batchJob {get; private set;}
+	private ID CsvStatRecId;
+	
+	public CW_AudienceCSVProcessController(){
+		CsvStatRecId=(ID) Apexpages.currentPage().getParameters().get('ID');
+		if(CsvStatRecId!=null){
+			impRevStat= [Select Schedule__c, Reward_Catalogue__c, NoRecords__c, NoRecToUpload__c, Name, MissingGroup__c, MissingFreeBet__c, MissingContact__c, Marketing_Campaign__c, Id From CW_CSV_Import_Status__c WHERE ID=:CsvStatRecId ];
+		}else{
+			//Rise error
+		}
+		countTiks=0;
+		refreshData();
+	}
+	
+	public Boolean getDisableActionPoller()
+	{
+		return batchJob.Status == ConstantUtil.BATCH_JOB_STATUS_COMPLETED 
+				|| batchJob.Status == ConstantUtil.BATCH_JOB_STATUS_ABORTED
+				|| batchJob.Status == ConstantUtil.BATCH_JOB_STATUS_FAILED;
+	}
+	
+	
+	
+	public void refreshData()
+	{
+		if(batchJobId==null){
+			batchJob=new AsyncApexJob();
+			IsProcessing=false;
+		}else{
+			integer lastJobItemsProcessed=0;
+			if(batchJob!=null){
+				lastJobItemsProcessed=batchJob.JobItemsProcessed;
+			}
+			List<AsyncApexJob> apxJbsLst=[select id,TotalJobItems,Status,NumberOfErrors,JobItemsProcessed,ExtendedStatus, 
+							CreatedDate, CompletedDate	from AsyncApexJob where id = :batchJobId];
+			if(apxJbsLst.Size()==0){
+				batchJob=new AsyncApexJob();
+				batchJobId=null;
+				IsProcessing=false;
+			}else{
+				batchJob=apxJbsLst[0];
+				IsProcessing=!getDisableActionPoller();
+			}
+			if(lastJobItemsProcessed!=batchJob.JobItemsProcessed)
+				countTiks=0;//clear 
+		}	
+		countTiks+=1;
+		if(CsvStatRecId!=null){
+			impRevStat= [Select Schedule__c, Reward_Catalogue__c, NoRecords__c, NoRecToUpload__c, Name, MissingGroup__c, MissingFreeBet__c, MissingContact__c, Marketing_Campaign__c, Id From CW_CSV_Import_Status__c WHERE ID=:CsvStatRecId ];
+		}
+		
+	}
+	
+	public void StartProcess(CSVBatchProcess.ProcesTypes ProcessID)
```

Comment:
Style issue: methods should use the "lowerCamelCase" notation. Suggestion: rename accordingly.